common/recordCreate.js

```diff
@@ -122,7 +122,10 @@
                         // happens with an error with code 0 (Timeout Error)
                         $log.warn(exception);
                         var message = exception.message || messageMap.errorMessageMissing;
-                        AlertsService.addAlert(message, 'error');
+                        // if online, we don't know how to handle the error
```

Comment:
What's the purpose of this?

Reply:
I made changes that look at the connection state of the browser to decide what the current error condition is. This  was the Alert that was being shown for the current error. Hongsuda wanted to handle this the same in all apps, so we have a modal dialog popup being shown (and this Alert is being suppressed).